src/pages/settings/schedules/common/atoms.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { Parameters } from 'common/interfaces/schedule';
+import { atom } from 'jotai';
+
+export const scheduleParametersAtom = atom<Parameters | undefined>(undefined);
```

Comment:
Generally, there's no need for new files for such things. It's okay to export them in the same file as the component where the need for such a thing is first.

Reply:
Okay, thank you for advise. Now, this atom is exported from EmailStatement component where it is used.